From daea88c572d8ef1d92b29ecf622eb11c138ab550 Mon Sep 17 00:00:00 2001
From: Maxim Karasev <begs@disroot.org>
Date: Wed, 8 Sep 2021 20:56:41 +0300
Subject: [PATCH] Simplify imv-folder script

This changes make it more portable by removing bash dependency and not
using GNU-specific 'sort' syntax. Also this fixes issue with selected
image not being displayed first.
---
 contrib/imv-folder | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/contrib/imv-folder b/contrib/imv-folder
index 33ea051..d9fb46f 100755
--- a/contrib/imv-folder
+++ b/contrib/imv-folder
@@ -1,10 +1,2 @@
-#!/bin/bash
-image="$1"
-declare -a ARGS order
-readarray -t -d '' order < <(printf '%s\0' "$(dirname "$image")"/* | sort -z --sort=version)
-for a in "${order[@]}"; do
-	if [ -f "$a" ]; then
-		ARGS+=("$a")
-	fi
-done
-exec imv "${ARGS[@]}" -n "$image"
+#!/bin/sh
+exec imv -n "$1" "$(dirname "$1")"
